src/compile/facet.ts

```diff
@@ -121,14 +121,21 @@ export class FacetModel extends Model {
     }, {});
   }
 
-  private initAxis(facet: Facet, child: Model): Dict<Axis> {
+  private initAxis(facet: Facet, config: Config, child: Model): Dict<Axis> {
     const model = this;
     return [ROW, COLUMN].reduce(function(_axis, channel) {
       if (facet[channel]) {
         const axisSpec = facet[channel].axis;
         if (axisSpec !== false) {
+          let vlAxisBase: VlOnlyAxisConfig = {};
```

Comment:
We no longer have VlAxisBase types, so you should rename all relevant variable names.